src/common/rules.js

```diff
@@ -130,6 +130,7 @@ YSLOW.registerRule({
         // array of regexps that match CDN Server HTTP headers
         servers: [
             'cloudflare-nginx' // not using ^ and $ due to invisible
```

Comment:
You're missing a trailing comma `,` at the end of this line

Reply:
Thanks for your feedback. We've submitted a new commit fixing this issue.

Reply:
Sorry @jaderhs, after I left my comment and put in my own PR for something different, I realized this repo is basically abandoned.